CMakeLists.txt

```diff
@@ -13,8 +13,13 @@ IF(WIN32)
   include_directories(${CMAKE_CURRENT_SOURCE_DIR}/contrib/win32)
   find_library(WINSOCK_LIB libws2_32.a)
   set(LIBS ${LIBS} ${WINSOCK_LIB})
+ELSE(WIN32)
+  set(_XOPEN_SOURCE 600)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/config_posix.h.cmake ${CMAKE_CURRENT_SOURCE_DIR}/config.h)
```

Comment:
`${CMAKE_CURRENT_SOURCE_DIR}/config.h` sould be `${CMAKE_BINARY_DIR}/config.h`.  This is wrong on line 12 too (we are not supposed to touch the source directory), please adjust both :wink: